### DISCUSSION TITLE
还是关于国足的事情

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1219] [提问] [answered]
联机什么时候好啊，服务器买不起吗？（SEVER好了，国足就打进世界杯了）
如果实在不行，拿给我钱，我买....

**Comment by 212112111zwl** [comment]
这不单单是买服务器的问题：

联机的界面要做吧~
开发出来总得测试测试吧~
内部测试完总得先给快照版用用吧~
况且每天issue一大堆，总不能只开发联机不管反馈吧~

**Comment by WForst-Breeze** [comment, marked as answer]
您在五天前问过一个相同问题，[已经有朋友为您解答了](https://github.com/Hex-Dragon/PCL2/discussions/1146#discussioncomment-4867672)，请您稍安勿躁（）至少近期应该不会完成，毕竟刚做完 Modrinth 下载，就单单这项功能必然需要一段时间的测试与稳定，一个个来，不着急>w<

**Comment by Creependerman** [comment]
所以联机没有采用去中心化的方式吗？（手动滑稽）

**Reply by Creependerman** [reply]
那这不会被wy制裁吧，服务器被封了就玩不了，搞去中心化只要启动器存在就能联（）

**Reply by Creependerman** [reply]
那联机功能3月左右就可以有了吧（

**Reply by Chiloven945** [reply]
> 那联机功能3月左右就可以有了吧（

没有捏（

**Comment by WorldHim** [comment]
不是既然已经问过就别重复发了吧。。。
而且这和国足有什么关系。。。

**Reply by WForst-Breeze** [reply]
只是个比喻（）这位朋友可能比较期待所以就很着急，可以理解（

**Comment by Enigma-Soul** [comment]
不是你想的

**Comment by Enigma-Soul** [comment]
...

**Comment by Enigma-Soul** [comment]
192.168.100.0就能

**Reply by WorldHim** [reply]
？看不懂

**Reply by Enigma-Soul** [reply]
虚拟路由器

**Reply by Enigma-Soul** [reply]
![image](https://user-images.githubusercontent.com/119654171/218090907-88142880-8acd-418d-bfea-44531e5b4f52.png)

**Reply by horizon86** [reply]
这得要公网ip才行，就是很多人没有才做联机功能。

**Reply by Enigma-Soul** [reply]
你很对

**Reply by Enigma-Soul** [reply]
但不完全对

**Reply by Enigma-Soul** [reply]
只要是个连inter的关猫都是有公网ip的，可以去路由设置查看

**Reply by horizon86** [reply]
> 只要是个连inter的关猫都是有公网ip的，可以去路由设置查看

并非，联通电信还能要公网ip，移动很多都是内网，现在联通电信有很多也是内网

**Reply by Enigma-Soul** [reply]
大多数是外网

**Reply by horizon86** [reply]
> 大多数是外网

要真是这样，那就没必要做启动器联机了，而事实是公网ipv4越来越收紧，v6还没有完全推广，你随便搜一下电信不开放公网ip的新闻，都有一堆。

**Comment by Enigma-Soul** [comment]
有免费的...路由器设置

**Comment by Enigma-Soul** [comment]
就是催更

**Comment by Pigeon0v0** [comment]
第一要处理许可证问题防止有心之人举办；

第二是网易那边可能有介入，即使是 P2P 也需要至少一个用于打洞的公网服务器，而且你还抢了人饭碗不是。

有些内容这里不好说，但是高度关注国内跨启动器联机的基本都知道了。

**Reply by Pigeon0v0** [reply]
@youzi-2333 @WForst-Breeze @Creependerman 

你们可以去关注一下最近发生了啥。

**Reply by 212112111zwl** [reply]
能不能说的具体一点，给个链接啥的（我也好奇

**Reply by Enigma-Soul** [reply]
对啊对啊

**Reply by Creependerman** [reply]
我知道啊，所以我交了个pr里面加了免责声明（

**Reply by Creependerman** [reply]
BakaXL作者被wyjb了

**Reply by Enigma-Soul** [reply]
挺惨的

**Reply by Enigma-Soul** [reply]
但是打洞这事网易应该管不着

**Reply by WForst-Breeze** [reply]
> 你们可以去关注一下最近发生了啥。

听说了（）挺可悲的……🤕

**Comment by Enigma-Soul** [comment]
建议龙猫把“M...”去掉

**Comment by so1ve** [comment]


**Reply by Enigma-Soul** [reply]
![image](https://user-images.githubusercontent.com/119654171/227078953-b804ac9d-2163-412e-b037-154cc358a3dc.png)

**Reply by so1ve** [reply]
6，喜欢帮龙猫写就去发PR 看你这心智也不太成熟：）

**Comment by WForst-Breeze** [comment]
请朋友们冷静，保持和谐友善讨论。一方面，我们都十分明白提问者对 PCL2 的期待和焦急的心情，请各位相互理解。另一方面，这个问题已经被回答了，一个月前的事就请尽量不要再旧事重提了。感谢各位的热情和对 PCL2 社区的支持和配合

**Reply by so1ve** [reply]
斯密马赛，我的问题（）删掉了
